CMakeLists.txt

```diff
@@ -140,26 +140,23 @@ if(BUILD_DOC)
 endif()
 
 find_package(PkgConfig)
-pkg_check_modules(SQLITE3 REQUIRED sqlite3)
-pkg_check_modules(BOTAN2 REQUIRED botan-2)
-pkg_check_modules(LIBUSB REQUIRED libusb-1.0)
-pkg_check_modules(GNUTLS REQUIRED gnutls)
+pkg_check_modules(sqlite REQUIRED IMPORTED_TARGET sqlite3)
+pkg_check_modules(botan-2 REQUIRED IMPORTED_TARGET botan-2)
+pkg_check_modules(libusb-1.0 REQUIRED IMPORTED_TARGET libusb-1.0)
+pkg_check_modules(gnutls REQUIRED IMPORTED_TARGET gnutls)
 
-find_package(CURL REQUIRED)
+find_package(Boost COMPONENTS locale REQUIRED)
```

Comment:
Since when are we using boost?

Reply:
Hm I see that boost::date_time is used in 
https://github.com/das-labor/neopg/blob/5ef973a0cbf763c496efb0a13bb0e3a582cdfbc4/lib/utils/time.cpp#L2
I will revert my boost date_time change.